src/algo_keys.c

```diff
@@ -1,17 +1,14 @@
+#include <string.h>
+
 #include "os.h"
 #include "cx.h"
 
 #include "algo_keys.h"
 
-// Allocate 64 bytes for privateKeyData because os_perso_derive_node_bip32
-// appears to write more than 32 bytes to this buffer.  However, we only
-// need 32 bytes for cx_ecfp_init_private_key..
```

Comment:
We should keep this comment somewhere -- it took a while to track down

Reply:
oh my mistake, didn't mean to delete this!